### ISSUE TITLE
SARIF Issue Severity Always INFO

### ISSUE THREAD

**Issue by JoshStutts** [tomasbjerre/violations-lib#144]
I'm using a couple different static analysis tools, both of which produce valid SARIF files according to https://sarifweb.azurewebsites.net/Validation

The SARIF files contain issues with their level set to "note", "warning", and "error", but the output from violations-lib after parsing a SARIF file only shows INFO severity items.

I've attached a sanitized SARIF file with my file paths removed and the extension changed to .txt so github would allow me to attach it, but it's still valid per the validator above.  Parsing this file using the violations-command-line results in the following:

![image](https://user-images.githubusercontent.com/2903366/159080390-99216db5-6816-41a6-9548-d1e6cb8c6ff5.png)

Actual levels in the file:
```
$ grep 'level":' security-scan.sarif | sort | uniq -c
   4           "level": "error",
   4           "level": "note",
  56           "level": "warning",
```

[security-scan.txt](https://github.com/tomasbjerre/violations-lib/files/8307541/security-scan.txt)

**Comment by tomasbjerre** [comment]
Should be fixed now in `1.21.3` of the command line tool.

**Comment by Olstyle** [comment]
I can confirm that this now works correctly for errors at least. Looks like the checker I use classifies everything as an error so I didn't see this when things I'd classify as info were marked like that.
